### PR TITLE
fix: Fix broken snapshot click event

### PR DESCRIPTION
**Pull request by ValGeorgiev**
Fix broken snapshot click event

image

### CODE REVIEW

File: packages/edition-slices/src/tiles/shared/tile-link.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const TileLink = ({
     article: { id, url }
   }
 }) => (
-  <Link linkStyle={style} onPress={() => onPress({ id, url })} url={url}>
+  <Link linkStyle={style} onPress={e => onPress(e, { id, url })} url={url}>
```
Comment (tuncaulubilge) [inline packages/edition-slices/src/tiles/shared/tile-link.js]: Can you do the same for puzzles as well, which has its own onPress event?

Comment (ValGeorgiev) [reply packages/edition-slices/src/tiles/shared/tile-link.js]: There is no need to add `event` object there. Puzzle onPress is using our custom `Link` component which is using `TouchableOpacity`. `TouchableOpacity` onPress doesn't require or use the event object.

Comment (tuncaulubilge) [reply packages/edition-slices/src/tiles/shared/tile-link.js]: Puzzle uses the same Link component as the TileLink. So we can remove the `event` object from both, and change the storybook so it doesn't depend on the `e` being passed

Comment (ValGeorgiev) [reply packages/edition-slices/src/tiles/shared/tile-link.js]: Yes, we can do that, but we need to preventDefault there

**Pull request merged**